api/tests/form-definitions.js

```diff
@@ -24,9 +24,6 @@ exports.forms = {
           tiny: 'RPY'
         },
         type: 'integer',
-        validate: {
-          is_numeric_year: true
```

Comment:
this, and other `is_numeric_xyz` functions are not actually implemented any more (except for month, which was implemented but not used by live projects and conflicted with co-located list declarations)

Reply:
also, this one says `validate` not `validations`